data/deep missions.txt

```diff
@@ -0,0 +1,2317 @@
+# Copyright (c) 2017 by Michael Zahniser
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+mission "Deep: Syndicate Convoy"
+	minor
+	name `Pickup from Hephaestus`
+	description `Escort a convoy of freighters to <stopovers> where they will pick up materials needed for scientific experiments. Return the convoy to <destination> by <date> for payment (<payment>).`
+	source
+		attributes "deep"
+		near "Gomeisa" 2
+	stopover "Hephaestus"
+	deadline
+	to offer
+		"combat rating" > 65
+		not "Deep: Mystery Cubes 4: done"
+		or
+			random < 20
+			and
+				"deep convoy" == 1
+				random < 30
+			and
+				"deep convoy" == 2
+				random < 40
+        
+	on offer
+		conversation
+			`You are approached by a man in a Deep Security uniform who asks if you would be willing to escort a few freighters to <stopovers> and back by <day>. "We would normally have our own fleet do it," he says, "but we're short on available ships given certain circumstances."`
```

Comment:
I recommend a slight tweak to the grammar here. Instead of "we're short on available ships _given_ certain circumstances", use "we're short on available ships _due to_ certain circumstances". The difference is that "given" usually indicates a precondition (eg. "given a spherical cow in a perfect vacuum"), whereas "due to" establishes cause and effect (eg. "this example is silly due to references to topologically improbably bovines").

Reply:
The improper grammar here can be attributed to the character's manner of speaking.